server/chat-plugins/trivia.ts

```diff
@@ -122,6 +122,22 @@ function isTriviaRoom(room: Room) {
 	return false;
 }
 
+function getTriviaGame(context) {
+	const room = Rooms.get('trivia');
+	if (!room) return false;
```

Comment:
```suggestion
	const room = context.room;
	if (!room) return false;
```

Reply:
(The original command wasn't scoped to the Trivia room, so this shouldn't be, either.)